src/js/util.js

```diff
@@ -96,8 +96,8 @@ var Util;
             return !!(obj && obj.nodeType === 1);
         },
 
-        now: function now() {
-            return Date.now() || new Date().getTime();
+        now: Date.now || function now() {
+            return +new Date();
```

Comment:
easy enough to remove them all ... either way this snippet is "fine" to land as it fixes a bug in the fallback implementation, and can later be removed outright.